<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PStyleChooser</class>
 <widget class="QWidget" name="PStyleChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QComboBox" name="comboBoxStyle">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>No Style</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Style</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonFG">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>f</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonBG">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>B</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonI">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>i</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
